Index: code.tsl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Output (Interlace (Read \"1.tl\") (Read \"1.tl\") )
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/code.tsl b/code.tsl
--- a/code.tsl	(revision dad8bbe70a93b51fee99245f1a43e096fa76a342)
+++ b/code.tsl	(date 1679085331648)
@@ -1,1 +1,1 @@
-Output (Interlace (Read "1.tl") (Read "1.tl") )
\ No newline at end of file
+Output (PlaceBelow (Read "1.tl") (Read "1.tl") )
\ No newline at end of file
